### PR TITLE
m-2553 - pmc-github-action-deployment

### PR DESCRIPTION
**Pull request by Or92**

### CODE REVIEW

**Review by liorzmetis — APPROVED** [review]

File: .github/workflows/promote-to-production.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          APP_ENV: production
+          INGEST_URL: ingest.metisdata.io
+          WEB_APP_HOST: app.metisdata.io
+        run: |
+          file_name=metadata-collector-cloudformation.yaml
+          cat ../../$file_name | \
+              sed -e "s+ingest.metisdata.io+$INGEST_URL+g" \
+                  -e "s+app.metisdata.io+$WEB_APP_HOST+g" \
+                  -e "s+Value: production+Value: $APP_ENV+g" \
+                  -e "s+Image: public.ecr.aws/o2c0x5x8/metis-md-collector:latest+Image: ${{ steps.retreive-latest-image-id.outputs.image_name }}+g" \
```
Comment (liorzmetis) [inline .github/workflows/promote-to-production.yml:41]: Is redundant you switch to the same values

File: .github/workflows/promote-to-production.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        with:
+          registry-type: public
+
+      - name: retreive the latest image id with latest-dev tag and repush it with latest tag
+        id: retreive-latest-image-id
+        env:
+          REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+        run: |
+          filter=*$REGISTRY/o2c0x5x8/metis-md-collector:latest-dev
+          IMAGE_ID=$(docker images --filter=reference=$filter -q | head -1)
```
Comment (liorzmetis) [inline .github/workflows/promote-to-production.yml:23]: set for each docker image also git sha commit and ask the user to set additional input that specify what commit to deploy to prod

File: .github/workflows/promote-to-production.yml (hunk 1 above)
Comment (Or92) [reply .github/workflows/promote-to-production.yml:41]: What do you mean what is redundant?

File: .github/workflows/promote-to-production.yml (hunk 2 above)
Comment (Or92) [reply .github/workflows/promote-to-production.yml:23]: Why the user should care about a commit? Once you click it you promote the latest to prod as the name implies

**Pull request merged**